src/user-guide/sharing-workflows/sharing-workflows.rst

```diff
@@ -0,0 +1,285 @@
+
+.. code-block:: python
+
+   c.CylcUIServer.site_authorization = {
+       "*": {  # For all ui-server owners,
+           "*": {  # Any authenticated user
+               "default": "READ",  # Will have default read-only access
+           },
+           "user1": {  # for all ui-server owners, user1
+               "default": ["!ALL"],  # has no privilidges by default
+           },  # No limit set, so all ui-server owners are unable to permit user1
+       },
+       "server_owner_1": {  # For specific UI Server owner, server_owner_1
+           "*": {  # Any authenticated user
+               "default": "READ",  # Will have default read-only access
+               "limit": ["READ", "CONTROL"],  # server_owner_1 is able to give away
+           },  # READ and CONTROL privileges.
+       },
+       "server_owner_2": {  # For specific UI Server owner,
+           "user2": {  # Specific user2
+               "limit": "ALL"  # Can only be granted a maximum of ALL by
+           },  # server_owner2, default access for user2 falls back to
+           # standard READ only (if server_owner_2/user2 are
+           # included in other auth config e.g. the top example),
+           # or none if not in any other auth config sections.
+           "group:groupA": {  # group denoted with a `group:`
+               "default": [
+                   "READ",
+                   "CONTROL",
+               ]  # groupA has default READ, CONTROL access to server_owner_2's
+           },  # workflows
+       },
+       "group:grp_of_svr_owners": {  # Group of users who own UI Servers
+           "group:groupB": {
+               "default": "READ",  # can grant groupB users up to READ and CONTROL
+               "limit": [  # privileges, without stop and kill
+                   "READ",
+                   "CONTROL",
+                   "!stop",  # operations
+                   "!kill",
+               ],
+           },
+       },
+   }
+
+
+.. _group_support:
+
+Group Support
+^^^^^^^^^^^^^
+Unix-like systems support grouping users. Cylc authorization supports granting
+access by membership of these system groups. You can indicate a system group
+by using the ``group:`` indicator.
+
+.. TODO - link to method getgroups() in uiserver, once Auth PR merged. 
+
+Changing Access Rights
+^^^^^^^^^^^^^^^^^^^^^^
+Changing authorization permissions in your `jupyter_config.py` will require the
+UI Server to be restarted before any changes are applied.
+
+Interacting with Others Workflows
+---------------------------------
+If you have been granted access to another users workflows, you can view and
+interact with these workflows.
+Say, you, userA, wishes to interact with userB's workflows.
+You can do this by changing the URL `https://<hub>/user/userB`, using the hub
+of the userB. You should authenticate as yourself (userA) and, provided you
+have the correct permissions, you will see userB's workflows for interaction.
+
```

Comment:
Not sure if screenshots would be appropriate here, so users know what to expect?